welcome/index.adoc

```diff
@@ -317,3 +317,18 @@ endif::[]
 endif::openshift-enterprise,openshift-webscale,openshift-origin[]
 
 endif::openshift-rosa[]
+
+== Hosted control plane activities
+
+* **Support for bare metal and {VirtProductName}**: Hosted control planes for {product-title} is now Generally Available on bare metal and {VirtProductName} platforms. For more information, see the following documentation:
+
+** link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.9/html/clusters/cluster_mce_overview#configuring-hosting-service-cluster-configure-bm[Configuring the hosting cluster on bare metal]
+** link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.9/html/clusters/cluster_mce_overview#hosted-control-planes-manage-bm[Managing hosted control plane clusters on bare metal]
+** link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.9/html/clusters/cluster_mce_overview#hosted-control-planes-manage-kubevirt[Managing hosted control plane clusters on OpenShift Virtualization]
+
+* **Technology Preview features**: Hosted control planes remains available as a Technology Preview feature on the Amazon Web Services platform, and is now available as a Technology Preview feature on the {ibmzProductName} platform. For more information, see the following documentation:
+
+** link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.9/html/clusters/cluster_mce_overview#hosting-service-cluster-configure-aws[Configuring the hosting cluster on AWS (Technology Preview)]
+** link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.9/html/clusters/cluster_mce_overview#configuring-hosting-service-cluster-ibmz[Configuring the hosted cluster on x86 bare metal for IBM Z compute nodes (Technology Preview)]
```

Comment:
```suggestion
** link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.9/html/clusters/cluster_mce_overview#configuring-hosting-service-cluster-ibmz[Configuring the hosted cluster on 64-bit x86 bare metal for IBM Z compute nodes (Technology Preview)]
```

Reply:
Based on the examples here, I think its supposed to be written as I've suggested. https://redhat-documentation.github.io/supplementary-style-guide/#bit-64-x86

Reply:
Fixed. Thanks!